KsApi/models/templates/CommentsEnvelopeTemplates.swift

```diff
@@ -37,4 +37,17 @@ extension CommentsEnvelope {
     slug: "slug",
     totalCount: 0
   )
+
+  internal static let successFailedRetryingRetrySuccessCommentsTemplate = CommentsEnvelope(
```

Comment:
Used this name `successFailedRetryingRetrySuccessCommentsTemplate` for two reasons
1. The flow is the status is actually `success` -> `failed` -> `retrying` -> `retryingSuccess` -> `success`
2. If I named it starting from prefix failed, I might probably want to name it `failedRetryingRetrySuccessSuccessCommentsTemplate `

Reply:
Hmm I see. I wonder if naming it something simple like `allStatesTemplate` might be more helpful to the reader?

Reply:
The comment has other states that are not included, so `allStatesTemplate` might be.
Maybe we you can help with a better name lol couldn't think of any that captures it.

Reply:
should be fine given the extra info you provided